src/test/scala/firrtlTests/ChirrtlMemSpec.scala

```diff
@@ -108,6 +108,23 @@ circuit foo :
     parse(res.getEmittedCircuit.value)
   }
 
+  "An mport that refers to an undefined memory" should "have a helpful error message" in {
+    val input =
+      """circuit testTestModule :
+         |  module testTestModule :
+         |    input clock : Clock
+         |    input reset : UInt<1>
+         |    output io : {flip in : UInt<10>, out : UInt<10>}
+         |
+         |    node _T_10 = bits(io.in, 1, 0)
+         |    read mport _T_11 = m[_T_10], clock
+         |    io.out <= _T_11""".stripMargin
+    assertThrows[PassException] {
+      (new LowFirrtlCompiler).compile(CircuitState(parse(input), ChirrtlForm), Seq()).circuit
```

Comment:
Can you check the message thrown? This introduces some brittleness on the exception message, but I think that's warranted versus aliasing of this to any `PassException`.

I think the following does it:
```scala
intercept[PassException]{ (new LowFirrtlCompiler).compile(CircuitState(parse(input), ChirrtlForm), Seq()).circuit }
  .getMessage should startWith ("Undefined memory m referenced by mport _T_11") // I think I have this right?
```

Reply:
I didn't know you could do that! Good thought